server/controllers/finance/reports/financial.patient.js

```diff
@@ -1,13 +1,13 @@
 /**
- * @overview reports/registrations
+ * @overview reports/finance/financePatient/{:uuid}
```

Comment:
Technically, the jsdoc `@overview` tag is for the file name.  This is fine, but might be confusing to someone who works with JSDoc a lot.

[Source](http://usejsdoc.org/tags-file.html).